Python/mergetwopdf/Readme.MD

```diff
@@ -0,0 +1,23 @@
+# Merge two pdf Files

```

Comment:
Rename the `mergetwopdf` to `merge_two_pdf` to make it look more readable and concise.